src/libcore/fmt/rt/v1.rs

```diff
@@ -33,9 +33,13 @@ pub enum Alignment {
     Unknown,
 }
 
+/// Used by [width](https://doc.rust-lang.org/std/fmt/#width) and [precision](https://doc.rust-lang.org/std/fmt/#precision) specifiers.
```

Comment:
This will link to the stable docs even from nightly.
```suggestion
/// Used by [width][core::fmt::width] and [precision][std::fmt::precision] specifiers.
```

Reply:
1. Would that expand to the right thing?
2. Is `cargo doc` ever run on this code? If not, I feel like a URL would be much easier to view than a path.

Reply:
1. Yes, see https://github.com/rust-lang/rust/issues/43466
2. Oh hmm I didn't realize this hole module uses `doc(hidden)`. In that case I guess it doesn't really matter, I was confused because it's technically public.

Reply:
1. Wouldn't it be `[crate::fmt#width]`, as per https://github.com/rust-lang/rust/pull/66675?
2. Huh, interesting. I guess it's so compiler-generated code can access it? (E.g. https://github.com/rust-lang/rust/blob/8534be72fc3b9c5f2f2dc4e4ee7b651a008e9a3e/src/librustc_builtin_macros/format.rs#L581)

Reply:
I don't see how that would expand to the right thing. Here is the error:

```console
error: `[crate::fmt::width]` cannot be resolved, ignoring it.
   --> src/libcore/fmt/rt/v1.rs:36:21
    |
36  | /// Used by [width][crate::fmt::width] and [precision][crate::fmt::precision] specifiers.
    |                     ^^^^^^^^^^^^^^^^^ cannot be resolved, ignoring
    |
note: the lint level is defined here
   --> src/libcore/lib.rs:155:9
    |
155 | #![deny(intra_doc_link_resolution_failure)]
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = help: to escape `[` and `]` characters, just add '\' before them like `\[` or `\]`

error: `[crate::fmt::precision]` cannot be resolved, ignoring it.
  --> src/libcore/fmt/rt/v1.rs:36:56
   |
36 | /// Used by [width][crate::fmt::width] and [precision][crate::fmt::precision] specifiers.
   |                                                        ^^^^^^^^^^^^^^^^^^^^^ cannot be resolved, ignoring
   |
   = help: to escape `[` and `]` characters, just add '\' before them like `\[` or `\]`
```

It works with `[width][crate::fmt#width]` but I agree with @sollyucko that a URL is fine.